File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        hash = {}

        for num in nums:
            hash[num] = hash.get(num, 0) + 1
        result = []
        for key, val in hash.items():
            if val > len(nums) // 3:
                result.append(key)
        return result

File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        Set<Integer> seen = new HashSet<>();

        for (int num: nums) {
            seen.add(num);
        }

        for (int i = 0; i < nums.length + 1; i++) {
            if (!seen.contains(i)) return i;
        }

        return -1;
        
    }
}
